src/errorcodes-net-cli/Validations/ValidateCommand.cs

```diff
@@ -16,8 +17,7 @@ internal class ValidateCommand : ICliCommand
 {
     private enum ValidationType
     {
-        CheckUniqueness,
-        Test
```

Comment:
Forgot to remove this before the last PR, oops!

Reply:
LGTM.